config/sync/views.view.publication_search_api.yml

```diff
@@ -0,0 +1,335 @@
+uuid: be58e360-d632-411d-aee4-89614cefa9d8
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_published_date
+    - search_api.index.publication_index
+  module:
+    - datetime
+    - search_api
+id: publication_search_api
+label: 'Publication search API'
+module: views
+description: ''
+tag: ''
+base_table: search_api_index_publication_index
+base_field: search_api_id
+core: 8.x
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: none
+        options: {  }
+      cache:
+        type: tag
+        options: {  }
+      query:
+        type: views_query
+        options:
+          bypass_access: false
+          skip_access: false
+          preserve_facet_query_args: false
+      exposed_form:
+        type: basic
+        options:
+          submit_button: Search
+          reset_button: false
+          reset_button_label: Reset
+          exposed_sorts_label: 'Sort by'
+          expose_sort_order: true
+          sort_asc_label: Asc
+          sort_desc_label: Desc
+      pager:
+        type: full
+        options:
+          items_per_page: 20
+          offset: 0
+          id: 0
+          total_pages: null
+          tags:
+            previous: '‹ Previous'
+            next: 'Next ›'
+            first: '« First'
+            last: 'Last »'
+          expose:
+            items_per_page: false
+            items_per_page_label: 'Items per page'
+            items_per_page_options: '5, 10, 25, 50'
+            items_per_page_options_all: false
+            items_per_page_options_all_label: '- All -'
+            offset: false
+            offset_label: Offset
+          quantity: 9
+      style:
+        type: default
+      row:
+        type: fields
```

Comment:
Your mileage may vary, but you may find that having views render entities in a specific view mode rather than selecting specific fields encourages consistency and involves less styling work if you already have those view modes defined.

Reply:
I'm a fan of Display modes for Views results and would use them in this instance  but I'd caution that it should be a judgement call and the main reason you would want to use fields is when you have a ton of data and multiple relationships, in which case you'd be loading multiple entities each time rather than using a much quicker database SQL query.